static/js/components/auth/TouchstoneLoginButton.js

```diff
@@ -4,10 +4,14 @@ import React from "react"
 
 import { TOUCHSTONE_URL } from "../../lib/url"
 
-const TouchstoneLoginButton = () => (
+type Props = {
+  next?: string
+}
+
+const TouchstoneLoginButton = ({ next }: Props) => (
   <a
     className="link-button outlined touchstone-text-logo"
-    href={TOUCHSTONE_URL}
+    href={next ? `${TOUCHSTONE_URL}?next=${next}` : TOUCHSTONE_URL}
```

Comment:
should be `${TOUCHSTONE_URL}&next=${next}`, a parameter of `?idp=default` is already included in `TOUCHSTONE_URL`